src/components/TablePerson/TablePerson.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+export const TablePerson: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames('', {
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames('', {
+            'has-text-danger': sex === 'f',
+          })}
+        >
+          {name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      {mother ? (
+        <td>
```

Comment:
In both cases you have `td`, only the content is different

Reply:
looks a little strange for me, but you are boss and I did it)